Tests/IntegrationTests.Shared/SimpleLINQtests.cs

```diff
@@ -40,16 +41,16 @@ public void CreateList()
         {
             var s0 = _realm.All<Person>().Where(p => p.Score == 42.42f).ToList();
             Assert.That(s0.Count(), Is.EqualTo(1));
-            Assert.That(s0[0].Score, Is.EqualTo(42.42f));
+            Assert.That(s0 [0].Score, Is.EqualTo(42.42f));
```

Comment:
Sigh, yes it looks like it crept back in at some point. I swear IDE settings randomly reset to default.